src/ViewArticle/components/VeiwArticleComponent.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import Ratings from '../../commons/assets/images/ratings.png';
+import Avarta from '../../commons/assets/images/avarta.jpg';
+import '../ViewArticleStyles/ViewArticle.scss';
+
+const ViewArticleComponent = ({ singleArticle }) => (
+  <div>
+    <div className="header">
+      <h3 className="article_title">{singleArticle.title}</h3>
+    </div>
+    <div className="flex-container">
+      <div className="top-div-inner custom-profile">
+        <div>
+          <img src={Avarta} className="avarta-image" alt="avarta" />
```

Comment:
@jennizalwango I am just wondering why the profile image of the user is static. I suggest you make this shown maybe when the article author has no profile image.

Reply:
Its because l am just trying to design how the page will be but the person working on profiles will be the one to implement that. Because my task is to just display a single article